### ISSUE TITLE
Close archive after take out process forcibly.

### ISSUE THREAD

**Issue by lyoutakoduka** [lyoutakoduka/spartaproject#95]
Currently if you take out multiple archives consecutively, archives aren't closed correctly at once.

https://github.com/lyoutakoduka/spartaproject/blob/6e101a1db0d2e7a7747590827339557978511548/pyspartaproj/script/file/archive/take_out_archive.py#L101

e.g., process to edit archives are like following step if you want to take out two archives.

1. Take out archive A
2. Take out archive B
3. Close archive A
4. Close archive B

Basically it should be like follow after update module.

1. Take out archive A
2. Close archive A
3. Take out archive B
4. Close archive B
